CHANGELOG.md

```diff
@@ -18,9 +18,11 @@ This can also be enabled programmatically with `warnings.simplefilter('default',
 
 ## [2.5.8] - not released yet
 ### Added
+- support for description list (`<dl>`), description titles (`<dt>`), and description details (`<dd>`).
```

Comment:
```suggestion
- support for description list (`<dl>`), description titles (`<dt>`), and description details (`<dd>`) in `write_html()`.
```

Reply:
apply the suggestion at 0dfbfaeb18a549c3e73cfacc51bc50708f2322c6